imap_processing/glows/l0/glows_l0_data.py

```diff
@@ -0,0 +1,153 @@
+class GlowsHistL0:
```

Comment:
Do you want to turn these into true DataClasses and keep the items accessed as attributes on the class `class.key` rather than via the `class.data[key]` dict lookup?
https://docs.python.org/3/library/dataclasses.html

Reply:
Yeah, that actually does make sense. Not sure why I didn't do that in the first place... but I will update it now!